src/vs/workbench/contrib/debug/browser/loadedScriptsView.ts

```diff
@@ -466,31 +466,37 @@ export class LoadedScriptsView extends ViewletPanel {
 			}
 		}));
 
-		const registerLoadedSourceListener = (session: IDebugSession) => {
+		const scheduleRefreshOnVisible = () => {
```

Comment:
Changes in the `loadedScriptsView` I leave up to @weinand to review.

Reply:
looks good to me...